### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by KamilKalitaRel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,24 @@
+*NOTE*: Relativity.Transfer.Client version 7.4.10 (and below) will stop functioning on Feb 28th, 2023 due to migration of data transfer related products to cloud based solutions. The new version of Relativity.Transfer.SDK will be available on Nov 30th 2022.  
+
+**Since it is going to be a breaking change you will need to update your code and integrate with the new the new Relativity.Transfer.SDK version – 1.0.0.**
```
Comment (KlaudiuszBryjaRelativity) [inline README.md]: We need to publish version 1.0.1 so maybe this version should be:
`SDK version – 1.0.*`
What do you think?

**Review by KlaudiuszBryjaRelativity — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (mmkowal-rel) [inline README.md]: typo ?  

> with the new the new

Comment (mmkowal-rel) [reply README.md]: what about the latest version

File: README.md [hunk 2]
```diff
@@ -1,3 +1,24 @@
+*NOTE*: Relativity.Transfer.Client version 7.4.10 (and below) will stop functioning on Feb 28th, 2023 due to migration of data transfer related products to cloud based solutions. The new version of Relativity.Transfer.SDK will be available on Nov 30th 2022.  
+
+**Since it is going to be a breaking change you will need to update your code and integrate with the new the new Relativity.Transfer.SDK version – 1.0.0.**
+
+November 30th 2022 we will release first version including:
+ * Upload a single file or a directory to the RelativityOne fileshare 
+ * Track data transfer progress
```
Comment (mmkowal-rel) [inline README.md]: add more details about succeeded, failed, skipped files including their source and destination paths

**Review by mmkowal-rel — APPROVED** [review]
Minor things

File: README.md [hunk 3]
```diff
@@ -1,3 +1,24 @@
...
+November 30th 2022 we will release first version including:
+ * Upload a single file or a directory to the RelativityOne fileshare 
+ * Track data transfer progress 
+ * Recover transfer from previous failure or interruption 
+ * Real time information about succeeded, failed and skipped items, including exact path and the reason of error 
+ * Cross platform compatibility - you can transfer data from Windows, Linux and macOS 
+ * Secured transfer using HTTPS, only single TCP 443 is required 
+
+Q1 2023 you can expect additional functionalities like: 
+ * Download a single file or a directory to the RelativityOne fileshare
```
Comment (mmkowal-rel) [inline README.md]: from

**Review by mmkowal-rel — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (KamilKalitaRel) [reply README.md]: Changed to 1.0.*

Comment (KamilKalitaRel) [reply README.md]: Redundant "the new" removed.

File: README.md (hunk 2 above)
Comment (KamilKalitaRel) [reply README.md]: Info added

File: README.md (hunk 3 above)
Comment (KamilKalitaRel) [reply README.md]: Fixed

**Pull request merged**